company-lsp.el

```diff
@@ -369,20 +369,25 @@ INCOMPLETE: t or nil. Whether the candidates are incomplete or not."
   "Get candidates from a CACHE-ITEM."
   (plist-get cache-item :candidates))
 
-(defun company-lsp--documentation (candidate)
+(defun company-lsp--documentation (candidate &optional only-markdown)
   "Get the documentation from the item in the CANDIDATE.
 
 The documentation can be either string or MarkupContent. This method
 will return markdown string if it is MarkupContent, original string
 otherwise. If the documentation is not present, it will return nil
-which company can handle."
+which company can handle.
+
+If ONLY-MARKDOWN is non-nil, it will return a documentation only if
+the string is a MarkupContent with `kind' set to `markdown'."
```

Comment:
nit: s/the string is a MarkupContent/the value is a hashtable representing MarkupContent/

MarkupContent is not a string :)